src/functionalTest/java/uk/gov/hmcts/reform/sscs/functional/handlers/createbundle/CreateBundleAboutToSubmitHandlerFunctionalTest.java

```diff
@@ -143,9 +143,9 @@ private void assertThatBundleStitchedSuccessfully(long caseId, int expectedBundl
         assertThat(updatedCaseDetails.getData().getCaseBundles())
             .hasSize(expectedBundles)
             .extracting(Bundle::getValue)
-            .allSatisfy(bundleDetails -> {
-                assertThat(bundleDetails.getStitchedDocument()).isNotNull();
-                assertThat(bundleDetails.getStitchStatus()).isEqualTo("DONE");
+            .allSatisfy(BundleDetails -> {
```

Comment:
I think this should be bundleDetails, otherwise it gives the impression that it's a static method